app/models/appointment_summary.rb

```diff
@@ -0,0 +1,30 @@
+class AppointmentSummary
+  include ActiveModel::Model
+
+  attr_accessor :supplementary_benefits, :supplementary_debt, :supplementary_ill_health,
+                :supplementary_defined_benefit_pensions, :appointment_type
+
+  %i(supplementary_benefits supplementary_debt
+     supplementary_ill_health supplementary_defined_benefit_pensions).each do |predicate_method|
```

Comment:
`predicate_method` as a name doesn't make sense here. They're attribute writers, not predicates.